app/api/user/views.py

```diff
@@ -0,0 +1,18 @@
+from uuid import UUID
+from fastapi import APIRouter, Depends
+from .schemas import APIKey, SearchResponseSchema
+
+router = APIRouter()
+
+
+@router.get("/{user_id}/key")
+async def get_user_api_key(
+    user_id: UUID):
+    
+    # Check Database if User id exist 
+    
+    # If user exist get user APIkey
+    userAPIKey = APIKey(key='3fa85f64-5740-2262-b3fc-2c963f36afa1', user='3fa85f64-5740-2262-b3fc-2c963f36afa1', continent_name='africa')
```

Comment:
I see you have continent_name here, that was a mistake made on the model diagram :sweat_smile:, please take it out

Reply:
Done